### PR TITLE
32 New icons + Appfilter updates 

### PR DESCRIPTION
**Pull request by Donnnno**
#2

- Caldender Import Export
- Earmouse
- Jamendo
- Myapplist

#129
- Offline Calendar

 #219 
- Metadata Remover

#185 
- Coinbase
- Duo Mobile
- Ebay
- Hulu
- Moto
- Moto Help
- Pokemon Go
- Audible
- Amazon Shopping
- Authenticate
- Canvas Student
- Car scanner
- Cashapp
- Golden 1
- Google fi
- Google WearOS
- Instacart
- Island
- Material Island
- Moto notifications
- Pebble
- Pebble dialer
- Starbucks
- Wand

Appfiter update for k9 mail.
and some appfiter updates

### CODE REVIEW

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -1043,6 +1101,7 @@
 
     <!-- K-9 Mail -->
     <item component="ComponentInfo{com.fsck.k9/com.fsck.k9.activity.Accounts}" drawable="k9mail" />
+    <item component="ComponentInfo{com.fsck.k9/com.fsck.k9.activity.MessageList}" drawable="k-9mail"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]: special characters aren't allowed here

Comment (Donnnno) [reply app/src/main/res/xml/appfilter.xml]: oops, fixed it

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -1136,9 +1195,14 @@
     <!-- Lawnchair -->
     <item component="ComponentInfo{ch.deletescape.lawnchair.plah/ch.deletescape.lawnchair.Launcher}" drawable="lawnchair" />
     <item component="ComponentInfo{ch.deletescape.lawnchair.plah/ch.deletescape.lawnchair.LawnchairLauncher}" drawable="lawnchair" />
+    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.LawnchairLauncher}" drawable="librechair"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]: Doesn't look like you added an icon for `librechair` or `librechairsettings`, did you mean to use the same ones as Lawnchair for those two?

Comment (Donnnno) [reply app/src/main/res/xml/appfilter.xml]: Yea Librechair is a fork of lawnchair with basically the same icons.

Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]:
```suggestion
    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.LawnchairLauncher}" drawable="lawnchair"/>
```

File: app/src/main/res/xml/appfilter.xml [hunk 3]
```diff
@@ -1136,9 +1195,14 @@
...
+
+    <!-- Lawnchair Notes -->
+    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.notes.NotesActivity}" drawable="standardnotes"/>
 
     <!-- Lawnchair Settings -->
     <item component="ComponentInfo{ch.deletescape.lawnchair.plah/ch.deletescape.lawnchair.settings.ui.SettingsLauncherActivity}" drawable="lawnchairsettings" />
+    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.settings.ui.SettingsLauncherActivity}" drawable="librechairsettings"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml]:
```suggestion
    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.settings.ui.SettingsLauncherActivity}" drawable="lawnchairsettings"/>
```

**Comment by dkanada** [comment]
By the way, if you still have the svg files for the new backgrounds you added and stuff, it would be great to add them in the metadata folder so we can modify them later.

**Comment by Donnnno** [comment]
cool, I've added the svg's

**Comment by Donnnno** [comment]
I think it's a web preview thing, it doesn't show up weird when I open it.

**Pull request merged**